instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -131,14 +172,26 @@ defmodule OpentelemetryPhoenix do
   end
 
   @doc false
-  def handle_endpoint_stop(_event, _measurements, %{conn: conn} = meta, _config) do
+  def handle_endpoint_stop(_event, _measurements, meta, _config) do
+    case adapter() do
+      :cowboy2 ->
+        :ok
```

Comment:
Why does only the non-cowboy2 version do anything?

Reply:
Because otel cowboy will close the span and set all the closing attributes.

Reply:
Oh. I see now. So this only starts a span if not using cowboy? I had thought a new span was started in the phoenix instrumentation.

Reply:
Correct. It will just continue the existing span which allows it to set the name properly and capture the full request.